idd/Energy+.idd.in

```diff
@@ -44025,7 +44025,13 @@ ZoneHVAC:LowTemperatureRadiant:VariableFlow,
         \key SurfaceFaceTemperature
         \key SurfaceInteriorTemperature
         \default MeanAirTemperature
-   A6 , \field Heating Design Capacity Method
+   A6,  \field Setpoint Control Type
+        \note (How setpoint temperature is defined)
+        \type choice
+        \key HalfFlowPower
+        \key ZeroFlowPower
```

Comment:
Hmm, I'll have to fully grok the use of these to determine if the naming is good.  It probably is, but they seem a little funny at first.

Reply:
Well, it's trying to deal with flow or power so that we can handle both the same way.  So, it's consistent between the variable flow and electric (both use the same key words).  Perhaps there's a better way, but I felt like this was reasonable.  If you feel there is something better, I can change it.

Reply:
OK, if those keywords are in place elsewhere, then let's call that precedent and leave them.  If they need to be adjusted, it can happen on both objects at a later time.  But probably not.